packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -111,7 +111,7 @@ function WorkflowPage() {
   const { data: verifiedNumbers } = trpc.viewer.workflows.getVerifiedNumbers.useQuery();
 
   useEffect(() => {
-    if (workflow && !isLoading) {
+    if (workflow && (workflow.steps.length === 0 || workflow.steps[0].stepNumber === 1)) {
```

Comment:
Temporary fix to make sure data is loaded correctly. Still need to find out why the order is wrong on first load when a step was added